src/current/v23.1/migration-overview.md

```diff
@@ -5,87 +5,304 @@ toc: true
+
+This approach enables you to take your time with the migration, and to pause or roll back as you [monitor the migration](#set-up-monitoring-and-alerting) for issues and performance. A phased rollout has the least risk and user impact, since you can control the blast radius of your migration by routing traffic for a subset of users or services. However, it has the highest development effort, since you will need to figure out how to migrate in phases while maintaining consistency.
+
+### Capacity planning
+
+Determine the size of the target CockroachDB cluster. To do this, consider your data volume and workload characteristics:
+
+- What is the total size of the data you will migrate?
+- How many active [application connections](recommended-production-settings.html#connection-pooling) will be running in the CockroachDB environment?
+
+Use this information to size the CockroachDB cluster you will create. If you are migrating to a {{ site.data.products.db }} cluster, see [Plan Your Cluster](../cockroachcloud/plan-your-cluster.html) for details:
+
+- For {{ site.data.products.dedicated }}, refer to the [example](../cockroachcloud/plan-your-cluster.html#dedicated-example) that shows how your data volume, storage requirements, and replication factor affect the recommended node size (number of vCPUs per node) and total number of nodes on the cluster.
+- For {{ site.data.products.serverless }}, your cluster will scale automatically to meet your storage and usage requirements. Refer to [Choosing resource limits](../cockroachcloud/learn-about-pricing.html#choosing-resource-limits) to learn about how to limit your resource consumption.
+- For guidance on sizing for connection pools, see the {{ site.data.products.db }} [Production Checklist](../cockroachcloud/production-checklist.html#use-a-pool-of-persistent-connections).
+
+If you are migrating to a {{ site.data.products.core }} cluster:
+
+- Refer to our [sizing methodology](recommended-production-settings.html#sizing) to determine the total number of vCPUs on the cluster and the number of vCPUs per node (which determines the number of nodes on the cluster).
+- Refer to our [storage recommendations](recommended-production-settings.html#storage) to determine the amount of storage to provision on each node.
+- For guidance on sizing for connection pools, see the {{ site.data.products.core }} [Production Checklist](recommended-production-settings.html#connection-pooling).
+
+### Application changes
+
+As you develop your migration plan, consider the application changes that you will need to make. These may relate to the following:
+
+- [Designing a schema that is compatible with CockroachDB.](#schema-design-best-practices)
+- [Creating effective indexes on CockroachDB.](#index-creation-best-practices)
+- [Handling transaction contention.](#handling-transaction-contention)
```

Comment:
Thanks for calling this out. I am going to file a separate ticket to track the relevant doc adjustments for Read Committed.

Reply:
We will still need this with read committed bc not everyone will switch to read committed. That being said, it will make this better. We can close this comment but plan to revisit this in the future.